Feat/Add meta to `neofs-lens`

carpawell opened the pull request:
Adds `meta` subcommand. Also, includes refactoring that separates `inspect` and `list` commands on `write-cache` and `blobovnicza` subcommands.

carpawell commented on cmd/neofs-lens/internal/meta/list-graveyard.go:
```diff
@@ -0,0 +1,49 @@
+		meta.WithEpochState(epochState{}),
+	)
+
+	common.ExitOnErr(cmd, common.Errf("could not open metabase: %w", db.Open(true)))
```
sure, added such func for all the components

cthulhu-rider commented on cmd/neofs-lens/internal/meta/root.go:
```diff
@@ -0,0 +1,32 @@
+type epochState struct{}
+
+func (s epochState) CurrentEpoch() uint64 {
+	return math.MaxUint64
```
Looks like a fake, what is this workaround for?

carpawell replied:
after #1634 the meta panics if no epoch state is presented. so that is just for returning all the objects in it

we can add a flag that specifies the current epoch but not sure if it is useful

cthulhu-rider reviewed (commented):
For future pls don't blend feature code with refactor

carpawell commented:
> For future pls don't blend feature code with refactor

Usually, I do not do PRs that way. That happened because of the order of my changes: first, i added the meta but then realized that it was incredibly hard to use it without a refactor (and separate the refactor from the feature). In fact, commits are quite independent now. 

Rebased CHANGELOG conflicts.
